controllers/storagecluster/rbd_mirror_debug.go

```diff
@@ -0,0 +1,129 @@
+// This is a temporary measure to enable verbose logging for RDR
+// This should be removed once RDR goes to GA.
+package storagecluster
+
+import (
+	"context"
+	"os"
+
+	ocsv1 "github.com/red-hat-storage/ocs-operator/api/v1"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/pointer"
+)
+
+func (r *StorageClusterReconciler) ensureRbdMirrorDebug(sc *ocsv1.StorageCluster) error {
+	if sc.Spec.Mirroring.Enabled {
+		return r.Client.Create(context.TODO(), rbdMirrorDebugJob(sc))
```

Comment:
Will this code path run each time the ocs operator reconciles? If the job already exists, there is no need to run it again.

Reply:
Okay, I will create a bool variable that stores if the job creation was already done, So we will call the create func
if (sc.Spec.Mirroring.Enabled  && !rbdMirrorDebugJobCreated), Sounds good?

Reply:
Instead of a new bool, ignoring the AlreadyExists error should actually handle it as Madhu suggested

Reply:
Cool, Will keep that